src/services/hmwk-service.js

```diff
@@ -1,90 +1,96 @@
-const initMondayClient = require('monday-sdk-js');
-
+const initMondayClient = require("monday-sdk-js");
 
 class HmwkService {
-
     static async getAllStudents(token, studentsBoardId) {
         try {
             const mondayClient = initMondayClient();
             mondayClient.setToken(token);
```

Comment:
I know the text-transformation example does this but...

I think it's better to move this code outside of `HmwkService`, e.g.

```js
const API_TOKEN = process.env.API_TOKEN;
const monday = require("monday-sdk-js")();
monday.setToken(API_TOKEN);

class HmwkService {
  static async getAllStudents(studentsBoardId) {
    ...
    await monday.api(...)
  }
```

_(code snippet adapted from https://github.com/mondaycom/monday-sdk-js#using-as-an-npm-module)_

That way, we don't have to pass in the `token` argument all the time, or reinitialize the monday client all the time.

How does that sound?